src/main/java/de/rwth/idsg/steve/config/SecurityConfiguration.java

```diff
@@ -61,11 +60,31 @@
  * @since 07.01.2015
  */
 @Slf4j
+@RequiredArgsConstructor
 @Configuration
 @EnableWebSecurity
 @Conditional(SteveProdCondition.class)
 public class SecurityConfiguration {
 
+    private final UserDetailsManager userDetailsManager;
+
+    @PostConstruct
+    public void postConstruct() {
+        String userName = SteveConfiguration.CONFIG.getAuth().getUserName();
+        if (userDetailsManager.userExists(userName)) {
```

Comment:
In fact, it can be skipped if an admin user already exists.

Reply:
see https://github.com/steve-community/steve/pull/1539/files#r1712974597

Reply:
If the migrated user from the config is deleted (replaced by a other web_user), doesn't the code recreate the user from the config at a restart of Steve? 
I would prefer skip if any admin is present in the db. In that case if someone accidentally deleted all web_user admins a simple restart recreates the standard admin

Reply:
okay, fair enough, done in https://github.com/steve-community/steve/pull/1539/commits/ffd7bca874db6306ad561936c92eda02d3fb3ab6